server/src/graphql/resolvers/queries.js

```diff
@@ -31,4 +31,8 @@ module.exports = {
       return invalidUser;
     }
   },
+
+  getAllRigs: async(parent) => {
+    return await db.Rig.findAll({ include: [db.Well] })
```

Comment:
could you wrap this in a try/catch block in case anything goes wrong?

Reply:
Whoops, forgot that part of your comment, here it is!